### PR TITLE
Implemented Sections 541 - 576

### PR DESCRIPTION
**Pull request by nnikolajj**
### What?
Implemented Section 541 - 576
### Why?
I have to do it :(
### How?
writing it in a yaml file
### Testing?
yes - nils json validation test
### Anything Else?
closes: #77 
closes: #101 
closes: #107 
closes: #116 
closes: #138 
closes: #154 
closes: #162 
closes: #208 
closes: #560
closes: #561
closes: #562
closes: #563
closes: #564
closes: #565
closes: #566
closes: #567
closes: #567 
closes: #574  
closes: #575  
closes: #576 
closes: #577 
closes: #579 
closes: #582
closes: #583  
closes: #584 
closes: #585 
closes: #586 
closes: #587 
closes: #590 
closes: #591 
closes: #592
closes: #871 
closes: #872   
closes: #873 
closes: #874 
closes: #875  
closes: #876

### CODE REVIEW

**Review by Noeell — APPROVED** [review]
Well done! 👏

File: fabbwled-backend/src/main/resources/sections/541-576.yaml [hunk 1]
```diff
@@ -0,0 +1,610 @@
+- number: 541
+  content:
+    - text: |
+        Burdened by the heavy chest full of coins, the three figures have
+        left deep footprints in the grass.
+        Make a SCOUTING roll at a Difficulty of 10.
+        Burdened by the heavy chest full of coins, the three figures have
+        left deep footprints in the grass.
+        Make a SCOUTING roll at a Difficulty of 10.
+    - choice:
```
Comment (oliver-jenny) [inline fabbwled-backend/src/main/resources/sections/541-576.yaml:10]: Scouting roll would be possible in Java

**Review by oliver-jenny — APPROVED** [review]
Lgtm

**Review by kaibria — APPROVED** [review]
lgtm

**Review by JJoeeyy — APPROVED** [review]
looks good

**Pull request merged**